### ISSUE TITLE
Open panel at cursor position

### ISSUE THREAD

**Issue by SUPERCILEX** [SUPERCILEX/gnome-clipboard-history#22]
From https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/issues/110

**Comment by rohmishra** [comment]
@SUPERCILEX  I wonder if  the accessibility features could be useful place to check but the IME OSK does recognize if the text input would be covered by the popup I guess by checking against height?

**Comment by rohmishra** [comment]
It checks the size of the whole window not the text space. Sorry; my bad. https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/ui/keyboard.js#L634

**Comment by SUPERCILEX** [comment]
I tried the mouse thing and ended up not liking it because the mouse wasn't in a predictable place. I'd be willing to add it behind an option, but I haven't figured out how to make `set_position` work reliably (right now it only works if the top bar is hidden).

**Comment by LubosRemplik** [comment]
:+1:

**Comment by orta** [comment]
My previous clipboard manager would show the window in the center of the screen, which is also a pretty reasonable answer to making it trivial to find

**Comment by LubosRemplik** [comment]
> My previous clipboard manager would show the window in the center of the screen, which is also a pretty reasonable answer to making it trivial to find

Exactly. with 5120 x 1440 screen i am using center position would be perfect

Tried to adjust things and compile myself, but no luck yet :)

**Comment by ruby232** [comment]
Was this feature implemented? I couldn't find it in the code, so I created my fork, and it seems to be working fine. Would you like me to create a merge request for this?

**Comment by SUPERCILEX** [comment]
Sure, I'd love to take a look (no guarantees I'll merge though).

**Comment by ruby232** [comment]
It only works when Guake is open, that's very strange but it doesn't throw any errors or warnings.

**Comment by TimFaro** [comment]
Any chance that this could be added? I'd be happy to have the window open in the center of the screen or be movable so it can be placed somewhere more convenient, I have two monitors and it's a bit tedious right now, otherwise it's the best clipboard manager imo

**Comment by Matheos96** [comment]
Was just about to create my own issue for this when I found this one. Disappointed it has been inactive for almost a year (more)... Any chance of improving this, or at least giving us an option to change in what monitor (or center) the window appears? I too have two monitors, and it would at least be way helpful if the window would appear in the active monitor, not always primary

**Comment by SUPERCILEX** [comment]
If anyone knows how to do this, feel free send over a PR.

**Comment by sepsol** [comment]
At least having the option to show it at the center of the active monitor (or the primary monitor) would be awesome; similar to how [Lightning GNOME Launcher Extension](https://gitlab.com/rimal.avimanyu/lightning-gnome-launcher-extension) does it.
